### PR TITLE
Hint reporting update

### PR DESCRIPTION
**Pull request by tabascq**
Two things:
1. Renamed "tickets" to tokens
2. Now showing tokens earned in the standings as well as tokens spent

### CODE REVIEW

**Comment by philipzloh** [comment]
> Renamed "tickets" to tokens

I wonder if the better plan is to call them coins, because that's what they are in code and also because "tokens" are a different thing that do appear in the UI

File: ServerCore/Pages/Events/Standings.cshtml [hunk 1]
```diff
@@ -35,9 +35,14 @@
                     </th>
                     @if (!Model.Event.HideHints)
                     {
+                        <th>
+                            <a asp-page="./Standings" asp-route-sort="@(Model.SortForColumnLink(StandingsModel.SortOrder.HintsEarnedAscending, StandingsModel.SortOrder.HintsEarnedDescending))">
```
Comment (philipzloh) [inline ServerCore/Pages/Events/Standings.cshtml:39]: Is the first entry the default sort when you sort by the column? I'd say descending for HintsEarned makes more sense

File: ServerCore/Pages/Events/Standings.cshtml.cs [hunk 2]
```diff
@@ -112,6 +112,12 @@ public async Task OnGetAsync(SortOrder? sort)
                 case SortOrder.ScoreDescending:
                     teamsFinal = teamsFinal.OrderByDescending(ts => ts.Score).ThenByDescending(ts => ts.Rank).ThenByDescending(ts => ts.Team.Name).ToList();
                     break;
+                case SortOrder.HintsEarnedAscending:
+                    teamsFinal = teamsFinal.OrderBy(ts => ts.Team.HintCoinsEarned).ThenBy(ts => ts.Rank).ThenBy(ts => ts.Team.Name).ToList();
```
Comment (philipzloh) [inline ServerCore/Pages/Events/Standings.cshtml.cs:116]: OrderBy ascending HintCoinsEarned (weaker to stronger team order)
ThenBy ascending Rank (stronger to weaker team order)

I think this file has a few of these ThenBys that break "conventional logic"

**Review by philipzloh — CHANGES_REQUESTED** [review]

File: ServerCore/Pages/Events/Standings.cshtml (hunk 1 above)
Comment (tabascq) [reply ServerCore/Pages/Events/Standings.cshtml]: Agree - fixed this and several others.

**Review by philipzloh — APPROVED** [review]
"more hints used" would usually map to "weaker team" in my mind, but no biggie

**Pull request merged**